components/script/dom/htmlmediaelement.rs

```diff
@@ -1770,6 +1770,7 @@ impl HTMLMediaElement {
 
                 // Step 5.
                 self.handle_resize(Some(metadata.width), Some(metadata.height));
+                self.upcast::<Node>().dirty(NodeDamage::OtherNodeDamage);
```

Comment:
If we move this into handle_resize (which is what I assume we care about dirtying the node for) we can remove a bunch of other explicit dirty calls from this file.